docs/dev/developer-guides/building-on-zksync/events.md

```diff
@@ -0,0 +1,40 @@
+# zkSync events
+
+## Overview 
+Given that smart contracts themselves cannot listen to any events, events are a mechanism to publish actions to listeners outside the blockchain.
```

Comment:
Events are a mechanism to publish information to listeners outside the blockchain, given that smart contracts themselves can't read them.